tests/graph/test_isomorphic.py

```diff
@@ -228,3 +228,84 @@ def test_same_degrees_non_isomorphic(self):
                 self.assertFalse(
                     retworkx.is_isomorphic(g_a, g_b, id_order=id_order)
                 )
+
+    def test_graph_vf2_mapping_identical(self):
+        graph = retworkx.generators.grid_graph(2, 2)
+        second_graph = retworkx.generators.grid_graph(2, 2)
+        mapping = retworkx.graph_vf2_mapping(graph, second_graph)
+        self.assertEqual(mapping, {0: 0, 1: 1, 2: 2, 3: 3})
+
+    def test_graph_vf2_mapping_identical_removals(self):
+        graph = retworkx.generators.path_graph(2)
+        second_graph = retworkx.generators.path_graph(4)
+        second_graph.remove_nodes_from([1, 2])
+        second_graph.add_edge(0, 3, None)
+        mapping = retworkx.graph_vf2_mapping(graph, second_graph)
+        self.assertEqual({0: 0, 1: 3}, mapping)
+
+    def test_graph_vf2_mapping_identical_removals_first(self):
+        second_graph = retworkx.generators.path_graph(2)
+        graph = retworkx.generators.path_graph(4)
+        graph.remove_nodes_from([1, 2])
+        graph.add_edge(0, 3, None)
+        mapping = retworkx.graph_vf2_mapping(
+            graph,
+            second_graph,
+        )
+        self.assertEqual({0: 0, 3: 1}, mapping)
+
+    def test_subgraph_vf2_mapping(self):
```

Comment:
There is also a different file for testing subgraph isomorphism functionality. Might be better to place this test there. https://github.com/Qiskit/retworkx/blob/main/tests/graph/test_subgraph_isomorphic.py